docs/reference/protocol-spec.md

```diff
@@ -516,7 +516,7 @@ However, low-level programming languages (such as C, for example) will likely re
 
 {{% alert title="Map pattern in RESP2" color="info" %}}
 RESP2 doesn't have a map type.
-Maps in RESP2 are represented by arrays, in which each element is a key-value tuple.
+Maps in RESP2 are represented by arrays, in which each two element is a key-value tuple.
```

Comment:
```suggestion
Maps in RESP2 are represented by arrays, in which each pair of elements is a key-value tuple.
```

Reply:
This is not correct.

A map in RESP2 is a flat array with alternating key and value, like `[key1, value1, key2, value2, ...]`. At least that's what HGETALL returns.

The next line "Each tuple is an array with two elements, these being the key and the value." is also wrong.

Reply:
The word "tuple" is confusing in particular. I will suggest another formulation.